HelpMyStreetFE/HelpMyStreetFE/ViewComponents/ViewLocationPopupViewComponent.cs

```diff
@@ -40,8 +40,8 @@ public async Task<IViewComponentResult> InvokeAsync(int jobId, CancellationToken
                 throw new UnauthorizedAccessException("No user in session");
             }
 
-            var jobSummary = await _jobCachingService.GetJobSummaryAsync(jobId, cancellationToken);
-            var canView = await _requestService.LogViewLocationEvent(user.ID, jobSummary.RequestID, jobId);
+            var jobSummary = jobId.HasValue ? await _jobCachingService.GetJobSummaryAsync(jobId.Value, cancellationToken) : (await _requestService.GetRequestDetailAsync(requestId.Value, user.ID, cancellationToken)).RequestSummary.JobSummaries.First(); ;
```

Comment:
We don't need the RequestDetail here.  Instead, use GetRequestSummaryAsync on the RequestCachingService.  (Sorry, many new services!!)